kustomize/overlays/test/config/propeller/resource_manager.yaml

```diff
@@ -0,0 +1,3 @@
+propeller:
```

Comment:
i guess the default should have been this - like in the test also? will speed up tests? But, then we want to test resource manager

Reply:
We only use it for qubole today and we do not test qubole in endtoend tests... our testing story needs some rethinking... but I agree we should have some integration tests that validate this is behaving correctly...